manifest.json

```diff
@@ -2,7 +2,7 @@
     "repos": [
         {
             "id": "tpos",
-            "organisation": "lnbits",
+            "organisation": "arcbtc",
```

Comment:
Are this changes ok? your this started in an old tpos repo, and got smuggled here?